newsletter.md

```diff
@@ -0,0 +1,14 @@
+
+During the past few months, the team has been working hard to make new features available in the OpenPaaS MVP release.
+
+
+The team implemented JMAP sharing, allowing one user to share a folder with another user of the same domain. The sharee gets the right, if allowed, to read, mark, delete and moves emails. This is especially useful to delegate the management of your emails to someone else. You can see a quick [video](https://www.youtube.com/watch?v=iKygmVKH-xU) presenting JMAP sharing feature in INBOX.
+
+
+We also implemented drafts. The email you compose can now be saved in INBOX, reopened, re-edited. Moving them to the Outbox mailbox will actually trigger the sending. You can see a quick [video](https://www.youtube.com/watch?v=iKygmVKH-xU) presenting the draft feature in INBOX.
```

Comment:
"Moving them to the Outbox mailbox will actually trigger the sending" : does this mean we manually move the email to the Outbox mailbox?

Reply:
No, we are not speaking about Inbox interface (we are the james team).

As of today sending an email with James is moving it to the Outbox using JMAP, it is not correlated to Inbox.

Reply:
Thanks @chibenwa